api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/revenuecatui/PaywallActivityLauncherAPI.kt

```diff
@@ -90,4 +106,11 @@ private class PaywallActivityLauncherAPI {
             true
         }
     }
+
+    fun checkPaywallDisplayCallback() {
+        @Suppress("EmptyFunctionBlock")
+        val paywallDisplayCallback = object : PaywallDisplayCallback {
+            override fun onPaywallShouldDisplay(shouldDisplayPaywall: Boolean) {}
```

Comment:
Not needed but wanted to suggest it just in case you think it's a good idea. What do you think about returning a `DisplayPaywallResult` in case we want to add more stuff to the result in the future?

Reply:
Hmm it's a good point... Do you have any idea of anything else we might want to return? I think right now a simple boolean suffices so I would prefer not to complicate the API without need.

Reply:
Sounds good!